src/common/reducers/repositories.js

```diff
@@ -1,42 +1,117 @@
+import merge from 'lodash/merge';
+import union from 'lodash/union';
+
 import * as ActionTypes from '../actions/repositories';
-import { parseGitHubRepoUrl } from '../helpers/github-url';
+
+// XXX move to entities.js
+// XXX should we be handling snaps here too?
+// no, because it's not part of the normalized result from github
+// so this should really be called github_entities
+// no, decompose instead ...
+export function entities(state = { snaps: {}, repos: {} }, action) {
+  if (action.payload && action.payload.entities) {
+    return merge({}, state, action.payload.entities);
+  }
+
+
+  const reposInitialState = {
+    __isSelected: false,
```

Comment:
I am not wedded to __format, but happy with whatever namespace distinguishes props from api and client props to avoid clobbering.